### PR TITLE
Profile form updates

### PR DESCRIPTION
**Pull request by jivey**
for: https://github.com/openstax/accounts/issues/1187

### CODE REVIEW

File: config/locales/pl.yml [hunk 1]
```diff
@@ -7,7 +7,17 @@ pl:
...
+    how_using_book_options:
+      - Jako główny podręcznik do mojego kursu
+      - Jako opcjonalny/zalecany podręcznik do mojego kursu
+      - Do nauczania, korepetycji lub wsparcia studentów poza ustawieniami kursu
+      - Dla mojej własnej wiedzy lub innej pracy
+    how_using_book_tail: "?"
+    num_students_using_book_base: Ile studentów korzysta z
+    num_students_using_book_time_period: każdego semestru?
+    num_students_using_book_supplemental: Uwzględnij sekcje prowadzone przez asystentów, którymi nadzorujesz
+    num_students_using_book_placeholder: Liczba studentów korzystających z tej książki na semestr
```
Comment (jivey) [inline config/locales/pl.yml:20]: @ewaforman 👋 If you have some time, would you mind giving these translations a look over? Here are the English versions: https://github.com/openstax/accounts/pull/1193/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R13 Thank you for your time!

Comment (jivey) [reply config/locales/pl.yml:20]: @ewaforman Thank you!

**Comment by jivey** [comment]
@RoyEJohnson If you have some time, would you mind reviewing the JS/CoffeeScript side of this PR?

**Review by mwvolo — APPROVED** [review]
Tested this locally and made sure everything was coming into Salesforce as expected. Looks good!

File: app/assets/stylesheets/newflow.scss [hunk 2]
```diff
@@ -143,6 +153,37 @@ input.has-error {
...
+  background:url("data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2000/svg' width='50px' height='50px'><polyline fill='" + $input-border-color +"' points='46.139,15.518 25.166,36.49 4.193,15.519'/></svg>") right no-repeat;
+  background-position: right 1rem center;
+  background-size: 18px 18px;
+  background-color: white;
+  border-radius: 0px;
+  padding: 1rem 0.8rem;
+  font-size: 16px;
+  height: 50px;
+  width: 100%;
+  // don't dispaly the selection arrow since we're using a SVG icon for it
```
Comment (RoyEJohnson) [inline app/assets/stylesheets/newflow.scss:167]: typo 'display'

**Review by RoyEJohnson — APPROVED** [review]

**Pull request merged**